blog/2024-11-mastering-effect-and-afterRenderEffect/README.md

```diff
@@ -0,0 +1,477 @@
+---
```

Comment:
The uppercase letters in the folder name are weird. can we change it to all lowercase?

```
2024-11-mastering-effect-and-afterrendereffect
```


or maybe even

```
2024-11-effect-afterrendereffect
```